pkgs/development/libraries/raft-canonical/default.nix

```diff
@@ -2,26 +2,31 @@
 
 stdenv.mkDerivation rec {
   pname = "raft-canonical";
-  version = "0.11.2";
+  version = "0.16.0";
 
   src = fetchFromGitHub {
     owner = "canonical";
     repo = "raft";
-    rev = "v${version}";
-    sha256 = "050dwy34jh8dihfwfm0r1by2i3sy9crapipp9idw32idm79y4izb";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-qDLAEhknY+BB83OC6tfi7w/ZY/K492J5ZglLNUoBwbc=";
   };
 
   nativeBuildInputs = [ autoreconfHook file pkg-config ];
   buildInputs = [ libuv lz4 ];
 
   enableParallelBuilding = true;
 
-  # Ignore broken test, likely not causing huge breakage
-  # (https://github.com/canonical/raft/issues/292)
+  # network tests all fail with child killeg by signal 6
+  patches = [
+    ./disable-net-tests.patch
+  ];
+
+  # Error: test/unit/test_uv_fs.c:51: assertion failed: _rv == RAFT_NOTFOUND (18 == 19)
+  # Error: child killed by signal 6 (Aborted)
   postPatch = ''
-    substituteInPlace test/integration/test_uv_tcp_connect.c --replace \
-      "TEST(tcp_connect, closeDuringHandshake, setUp, tearDownDeps, 0, NULL)" \
-      "TEST(tcp_connect, closeDuringHandshake, setUp, tearDownDeps, MUNIT_TEST_OPTION_TODO, NULL)"
+    substituteInPlace test/unit/test_uv_fs.c --replace \
+      "TEST(UvFsCheckDir, doesNotExist, DirSetUp, DirTearDown, 0, NULL)" \
+      "TEST(UvFsCheckDir, doesNotExist, DirSetUp, DirTearDown, MUNIT_TEST_OPTION_TODO, NULL)"
```

Comment:
This test only fails on x86_64 for me, but passes on aarch64. Which is unfortunate, because the test suite runs the test anyway on and fails if it passes!